src/autotype/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 if(WITH_XC_AUTOTYPE)
-    if(UNIX AND NOT APPLE)
+    if(UNIX AND NOT APPLE AND NOT HAIKU)
```

Comment:
Same here ```if(UNIX AND NOT (APPLE OR HAIKU))```

Reply:
That makes no sense. They are logically equivalent.

Reply:
Making this change allows MacOS to build again, so I disagree (or at least CMake does)

Reply:
nevermind, I jumped the gun. CI failed for some unknown reason